# **Sales Force Training**

**This project aims to evaluate the impact of a sales training program on the average sales per transaction. From historical data, the company observed that the average sales was $100 per transaction. After implementing a training program for the sales team, a sample of 25 sales transactions was collected to assess whether the training led to a significant change in performance.**

In [21]:
# import packages
import pandas as pd
import numpy as np
import statistics

In [22]:
# making a dataframe
data_number= [100,150,50,100,130,120,100,85,70,150,150,120,50,100,100,140,90,150,50,90,120,100,110,75,65]

In [23]:
df=pd.DataFrame({"TransactionAmount":data_number})
df



TransactionAmount
0                 100
1                 150
2                  50
3                 100
4                 130
5                 120
6                 100
7                  85
8                  70
9                 150
10                150
11                120
12                 50
13                100
14                100
15                140
16                 90
17                150
18                 50
19                 90
20                120
21                100
22                110
23                 75
24                 65

In [24]:
data=np.array(df)
data

array([[100],
       [150],
       [ 50],
       [100],
       [130],
       [120],
       [100],
       [ 85],
       [ 70],
       [150],
       [150],
       [120],
       [ 50],
       [100],
       [100],
       [140],
       [ 90],
       [150],
       [ 50],
       [ 90],
       [120],
       [100],
       [110],
       [ 75],
       [ 65]])

In [25]:
# measure of central tendency 
mode=statistics.mode(df['TransactionAmount'])
print("mean:", np.mean(data))
print("median:", np.median(data))
print("mode:", mode)

# measure of variability of spread
print("var:", np.var(data))
print("std:", np.std(data))
print("range:", np.max(data)-np.min(data))
print("q1:", np.quantile(data,0.25))
print("q2:", np.quantile(data,0.50))
print("q3:", np.quantile(data,0.75))
print("IQR", np.quantile(data,0.75)-np.quantile(data,0.25))

mean: 102.6
median: 100.0
mode: 100
var: 972.2399999999999
std: 31.18076330047101
range: 100
q1: 85.0
q2: 100.0
q3: 120.0
IQR 35.0


In [26]:
#T-test
import scipy.stats as stats

In [27]:
data=np.array([100,150,50,100,130,120,100,85,70,150,150,120,50,100,100,140,90,150,50,90,120,100,110,75,65])
data

array([100, 150,  50, 100, 130, 120, 100,  85,  70, 150, 150, 120,  50,
       100, 100, 140,  90, 150,  50,  90, 120, 100, 110,  75,  65])

In [28]:
# state hyphothesis
#H0: Transaction Amount Average = 100
#H1: Transaction Amount Average > 100



In [29]:
# Define alpha=0.05
# Calculate T-Statistics and Perform One Sample T-test

stats.ttest_1samp(data, popmean=100)


TtestResult(statistic=np.float64(0.4085001556802841), pvalue=np.float64(0.6865284813438117), df=np.int64(24))

## **Statistic= 0.4085001556802841, p-value= 0.6865284813438117m**

- **Because the p-value is greater than alpha (0.05), we lack sufficient evidence to reject H0.**
- **Since the p-value is greater than to the significance level of 5%, we fail to reject H₀.**
- **This means there is not enough statistical evidence to conclude that the average sales increased after the training.**
- **Thus, it cannot be concluded that the sales force training had a significant impact on increasing sales.**